.github/workflows/dump-ci-stats.yml

```diff
@@ -26,6 +26,9 @@ jobs:
       - name: Set up Node.js
         uses: actions/setup-node@v4
 
+      - name: Set up yarn
+        run: corepack enable
+
       - name: Install GCP Monitoring/Metrics Client
         run: yarn add @google-cloud/monitoring --ignore-workspace-root-check
```

Comment:
Now the new `yarn` fails if given the `--ignore-workspace-root-check` flag. 😦 

Hopefully that's the last error.